tools/db_crashtest.py

```diff
@@ -422,6 +422,12 @@ def whitebox_crash_main(args, unknown_args):
                 "ops_per_thread": cmd_params['ops_per_thread'],
                 "compaction_style": 1,
             }
+            # Single level universal has a lot of special logic. Ensure we cover
+            # it sometimes.
+            if random.randint(0, 1) == 1:
+                additional_opts.update({
+                    "num_levels": 1,
```

Comment:
Do we need to migrate the data if there is existing data in non-L0 levels for this case?

Reply:
huh, I assumed that happened automatically, as in H2 of the whitebox crash test, it alternates between universal and FIFO compaction. So I thought it'd have to clear the DB between runs...

Reply:
How about set `destroy_db_initially` to true for these cases that change compaction style? At least until we come up with something better than cycling through the rare compaction strategies during the second half of crash test...